ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -37,7 +37,16 @@ cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
 mkdir -p ~/.ssh
 cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
-openshift-install --dir="${dir}" create manifests &
+if [[ "${USE_ETCD_RAMDISK}" == "true" ]]; then
+  openshift-install --dir=${dir} create ignition-configs
+  python -c \
+      'import json, sys; j = json.load(sys.stdin); j[u"systemd"] = {}; j[u"systemd"][u"units"] = [{u"contents": "[Unit]\nDescription=Mount etcd as a ramdisk\nBefore=local-fs.target\n[Mount]\n What=none\nWhere=/var/lib/etcd\nType=tmpfs\nOptions=size=2G\n[Install]\nWantedBy=local-fs.target", u"enabled": True, u"name":u"var-lib-etcd.mount"}]; json.dump(j, sys.stdout)' \
+      <${dir}/master.ign \
+      >${dir}/master.ign.out
+  mv ${dir}/master.ign.out ${dir}/master.ign
+fi
+
+openshift-install --dir=${dir} create manifests &
```

Comment:
The manifests should be created before the ignition configs.

Reply:
done